csharp/Platform.Collections/Lists/IListExtensions.cs

```diff
@@ -226,9 +237,9 @@ public static bool EqualTo<T>(this IList<T> left, IList<T> right, Func<IList<T>,
         /// <para>Compares each element in the list for identity.</para>
         /// <para>Сравнивает на идентичность каждый элемент списка.</para>
         /// </summary>
+        /// <typeparam name="T"><para>The list's item type.</para><para>Тип элементов списка.</para></typeparam>
         /// <param name="left"><para>The checked list.</para><para>Список для проверки.</para></param>
         /// <param name="right"><para>The checked list.</para><para>Список для проверки.</para></param>
-        /// <typeparam name="T"><para>The list's item type.</para><para>Тип элементов списка.</para></typeparam>
         /// <returns>
         /// <para>If every element of one list is not equal to every element of another list - return false, otherwise - true.</para>
```

Comment:
`If at least one element of one list is not equal to the corresponding element from another list returns false, otherwise - true.`